crates/locutus-stdlib/src/interface.rs

```diff
@@ -78,7 +78,26 @@ impl UpdateModification {
 
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq, Default)]
 pub struct RelatedContracts {
-    map: HashMap<ContractInstanceId, Option<State<'static>>>,
+    pub map: HashMap<ContractInstanceId, Option<State<'static>>>,
```

Comment:
this should be at much `pub(crate)` , but not compeltely public sicne we want to hide impl details

Reply:
done